other/upgrade.php

```diff
@@ -2725,6 +2729,27 @@ function cmdStep0()
 		require_once($modSettings['theme_dir'] . '/languages/Install.' . $upcontext['language'] . '.php');
 	}
 
+	// Do we need to add this setting?
+	$need_settings_update = empty($modSettings['custom_avatar_dir']);
+
+	$custom_av_dir = !empty($modSettings['custom_avatar_dir']) ? $modSettings['custom_avatar_dir'] : $GLOBALS['boarddir'] . '/custom_avatar';
+	$custom_av_url = !empty($modSettings['custom_avatar_url']) ? $modSettings['custom_avatar_url'] : $boardurl . '/custom_avatar';
+
+	// This little fellow has to cooperate...
+	quickFileWritable($custom_av_dir);
+
+	// Are we good now?
+	if (!is_writable($custom_av_dir))
+		return throw_error(sprintf($txt['error_dir_not_writable'], $custom_av_dir));
```

Comment:
Missing global

Reply:
Which global?   

FYI - this corrected the issue in actual testing.

Reply:
Looks like $txt

Reply:
Yep, will fix.

Reply:
In hindsight, this should be made more consistent with the error processing in that function, which is CLI specific.